org.aposin.licensescout.core/src/main/java/org/aposin/licensescout/exporter/TxtExporter.java

```diff
@@ -73,11 +73,10 @@ public OutputFileType getOutputFileType() {
     @Override
     public void export(final OutputResult outputResult, final ReportConfiguration reportConfiguration)
             throws Exception {
-        final String nl = "\n";
+        final String nl = System.lineSeparator();
```

Comment:
Does it make sense to create a static constant at the top-level class to re-use in this method and the other one changed below?

Reply:
As the TxtExporter has to be reworked completely anyway (see https://github.com/aposin/LicenseScout/issues/33), I suggest to leave this as it is for now.

Reply:
👍